boards/FSM/FSM.ioc

```diff
@@ -2,8 +2,9 @@
 ADC2.Channel-0\#ChannelRegularConversion=ADC_CHANNEL_1
 ADC2.Channel-1\#ChannelRegularConversion=ADC_CHANNEL_3
 ADC2.DMAContinuousRequests=ENABLE
+ADC2.EOCSelection=ADC_EOC_SINGLE_CONV
 ADC2.ExternalTrigConv=ADC_EXTERNALTRIGCONV_T3_TRGO
-ADC2.IPParameters=Rank-0\#ChannelRegularConversion,Channel-0\#ChannelRegularConversion,SamplingTime-0\#ChannelRegularConversion,OffsetNumber-0\#ChannelRegularConversion,Offset-0\#ChannelRegularConversion,NbrOfConversionFlag,ExternalTrigConv,NbrOfConversion,DMAContinuousRequests,Rank-1\#ChannelRegularConversion,Channel-1\#ChannelRegularConversion,SamplingTime-1\#ChannelRegularConversion,OffsetNumber-1\#ChannelRegularConversion,Offset-1\#ChannelRegularConversion,SubFamily
+ADC2.IPParameters=Rank-0\#ChannelRegularConversion,Channel-0\#ChannelRegularConversion,SamplingTime-0\#ChannelRegularConversion,OffsetNumber-0\#ChannelRegularConversion,Offset-0\#ChannelRegularConversion,NbrOfConversionFlag,ExternalTrigConv,NbrOfConversion,DMAContinuousRequests,Rank-1\#ChannelRegularConversion,Channel-1\#ChannelRegularConversion,SamplingTime-1\#ChannelRegularConversion,OffsetNumber-1\#ChannelRegularConversion,Offset-1\#ChannelRegularConversion,SubFamily,EOCSelection
```

Comment:
what is this extra `EOCSelection` for?

Reply:
Not sure. I'm assuming this was added when I changed ADC2's EOC back to single conversion as per previous comment.